interfaces/cuda/Control.cu

```diff
@@ -140,6 +140,12 @@ unsigned ConcreteAPI::getGlobMemAlignment() {
   return 128;
 }
 
+size_t ConcreteAPI::alignBytes(size_t requestedBytes) {
+  auto alignment = getGlobMemAlignment();
+  size_t alignmentFactor = (requestedBytes + alignment - 1) / alignment;
```

Comment:
Why the `+ alignment - 1`?

Reply:
You search the smallest number which is larger or equal than `requestedBytes` and a multiple of `alignment`. 
So for any `i`, we map `i*alignment+1`, ..., `i*alignment + alignment` to `(i+1) * alignment`. That's what this formula does.

Reply:
ok thx.